control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -1725,7 +1730,98 @@ impl From<&dyn SetVolumeReplicaInfo> for SetVolumeReplicaRequest {
         }
     }
 }
+/// Trait to be implemented for SetVolumeProperty operation.
+pub trait SetVolumePropertyInfo: Send + Sync + std::fmt::Debug {
+    /// Uuid of the concerned volume
+    fn uuid(&self) -> VolumeId;
+    /// Property to be set for the volume
```

Comment:
End doc comments with `.`
```suggestion
    /// Uuid of the concerned volume.
    fn uuid(&self) -> VolumeId;
    /// Property to be set for the volume.
```